Modules/_collectionsmodule.c

```diff
@@ -2546,24 +2546,59 @@ static PyTypeObject tuplegetter_type = {
 
 /* module level code ********************************************************/
 
-PyDoc_STRVAR(module_doc,
+PyDoc_STRVAR(collections_doc,
 "High performance data structures.\n\
 - deque:        ordered collection accessible from endpoints only\n\
 - defaultdict:  dict subclass with a default value factory\n\
 ");
 
-static struct PyMethodDef module_functions[] = {
+static struct PyMethodDef collections_methods[] = {
     _COLLECTIONS__COUNT_ELEMENTS_METHODDEF
     {NULL,       NULL}          /* sentinel */
 };
 
+static int
+collections_exec(PyObject *m) {
```

Comment:
nitpick: would you mind to rename it to "mod" or "module"?

Reply:
module might be great! Thanks for the suggestion